### ISSUE TITLE
Documentation on how to run web export is missing info on CORS headers and giving a sample python script that doesn't work

### ISSUE THREAD

**Issue by liana-p** [godotengine/godot-docs#8019]
**Your Godot version:** 4.1.1

**Issue description:**

This is about the [Exporting for web](https://docs.godotengine.org/en/stable/tutorials/export/exporting_for_web.html) documentation.

* The page mentions that serving the files requires CORS headers because of `SharedArrayBuffer`, but doesn't make it clear which ones or how to do it
* The page gives a tip to use a Python script to host exported games locally for testing, but the script doesn't work. When I try using it, pages don't load.

**URL to the documentation page (if already existing):** https://docs.godotengine.org/en/stable/tutorials/export/exporting_for_web.html

Opened PR #8020 with a fix for those two issues by replacing the python script with a different example local testing server setup.

**Comment by jynus** [comment]
Hello, liana-p,

thanks for the report. I agree with the first part of the report- Having explicit headers rather than just a link would be clearer. For example, having a section on how to setup that on different http servers.

Regarding the second part- the script you refer to, as it is now, works for me- and I was able to play a game exported with Godot 4 (the script includes [those headers](https://github.com/godotengine/godot/blob/master/platform/web/serve.py#L12)!). I wonder what problem you encountered (what errors or blockers you got and what platform you tried to run it on), and if it would be preferred to fix that issue rather than linking to an external (from Godot's point of view) repository.

Or, in the case a node.js over python solution was preferred, if it shouldn't live in the godot repository instead, replacing or alongside, the current python solution, and uploading the [server file](https://github.com/liana-p/godot4-web-local-testing-example/blob/main/export/index.js) directly there?

**Comment by jynus** [comment]
In particular, there used to be a node solution for the http server, but [it was replaced by the current python one](https://github.com/godotengine/godot/commit/1be1b15a57eb74d3c8d56683e6a221fd4712915c) for the reasons given in that commit.

**Comment by liana-p** [comment]
> In particular, there used to be a node solution for the http server, but [it was replaced by the current python one](https://github.com/godotengine/godot/commit/1be1b15a57eb74d3c8d56683e6a221fd4712915c) for the reasons given in that commit.

Having both would be good to have an alternative, I just wasn't sure about editing code in other godot repos, and since I had made this I figured I'd link it in the docs.

As for the python one, I'm not sure why but pages never loaded when I used it. Maybe it's just my machine. I was on Windows, python 3.11.5, tried Firefox and Chrome. Then I tried another python script I found online, and despite having the right CORS errors it had MIME file type errors... then I tried a simple node.js server and it just wortked.

For the PR, I could always edit it to have just the part about the headers and delete the rest for now
